proposals/vector_to_points.json

```diff
@@ -0,0 +1,68 @@
+{
+    "id": "vector_to_points",
+    "summary": "Sample random points from geometries",
+    "description": "For each geometry samples a number of random points based on one of the sampling method.",
+    "categories": [
+        "cubes",
+        "vector"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "Input geometries for sample extraction.\n\nTo maximize interoperability, a nested `GeometryCollection` should be avoided. Furthermore, a `GeometryCollection` composed of a single type of geometries should be avoided in favour of the corresponding multi-part type (e.g. `MultiPolygon`).",
+            "schema": [
+                {
+                    "type": "object",
+                    "subtype": "geojson"
+                },
+                {
+                    "type": "object",
+                    "subtype": "vector-cube"
+                }
+            ]
+        },
+        {
+            "name": "count",
+            "description": "The number of points to compute. For each point given the maximum number of computed points is always 1.",
```

Comment:
I would add something like "desired" in between: "The desired number of points to compute. ...". It is not obvious that the process can extract exactly that specific amount of points.
An edge case: we provide 5 points and ask for 10: not possible as we said that for each point we can extract only one.

Or maybe I'm just thinking too much and it's not necessary.

Reply:
maybe we can consider also to just name it "n" (generally used as placeholder) or "size" (used e.g. by Edzer in his sf package). "Count" is more likely used in statistics to count certain occurrences

Reply:
Sure: https://github.com/Open-EO/openeo-processes/pull/315/files#r782296286